.github/workflows/seo-audit.yml

```
@@ -0,0 +1,76 @@
+name: Landing Page Audit Pipeline
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)